pkg/dao/types/resource_revision.go

```diff
@@ -9,7 +9,7 @@ import (
 
 type OutputValue struct {
 	Name      string          `json:"name,omitempty"`
-	Value     property.Value  `json:"value,omitempty"`
+	Value     property.Value  `json:"value,omitempty,cli-table-column"`
 	Type      cty.Type        `json:"type,omitempty"`
 	Sensitive bool            `json:"sensitive,omitempty"`
```

Comment:
Do we need to expose sensitive as well like in variable?

Reply:
Now the outputs API doesn't return the sensitive field, it uses <sensitive> to replace the value. So we customize the variable list, and replace value with <sensitive> for sensitive value.
Variable list:
<img width="684" alt="image" src="https://github.com/seal-io/walrus/assets/5936271/adcac8ce-34e1-41bf-ab8b-bb497b5ac848">

Output list:
<img width="676" alt="image" src="https://github.com/seal-io/walrus/assets/5936271/18ddc57c-be61-484d-9836-c68dbb239fbf">